language/en_us/paystack.php

```diff
@@ -4,6 +4,7 @@
 
 
 $lang['Paystack.name'] = 'Paystack Standard';
+$lang['Paystack.description'] = 'Paystack is the fastest, simplest way to start accepting online payments in Nigeria. From signup to receiving real payments can take under 15 minutes. Seriously';
```

Comment:
Let's just update this to "Paystack is the fastest, simplest way to start accepting online payments in Nigeria."
i.e., drop the last 2 sentences

Reply:
Updated